net-mgmt/collectd/src/opnsense/mvc/app/models/OPNsense/Collectd/General.xml

```diff
@@ -141,5 +141,19 @@
             <default>0</default>
             <Required>N</Required>
         </p_ipstats>
+	<p_exec type="BooleanField">
+            <default>0</default>
+            <Required>N</Required>
+        </p_exec>
+	<p_exec_user type="TextField">
+            <default>daemon</default>
+            <Required>N</Required>
+	    <mask>/^[a-z_]([a-z0-9_-]{0,31}|[a-z0-9_-]{0,30}\$)$/</mask>
+        </p_exec_user>
+	<p_exec_script type="TextField">
+            <default></default>
+            <Required>N</Required>
```

Comment:
I'm not sure if there's a reasonable way to make this required if the enable-plugin box is checked?